src/meta/src/hummock/manager/time_travel.rs

```diff
@@ -483,14 +487,8 @@ fn replay_archive(
     deltas: impl Iterator<Item = PbHummockVersionDelta>,
 ) -> HummockVersion {
     let mut last_version = HummockVersion::from_persisted_protobuf(&version);
-    let mut mce = last_version.visible_table_committed_epoch();
```

Comment:
Actually we can still assert `mce >= prev_mce` to ensure that the mce won't decrease.